pennylane/qnn/keras.py

```diff
@@ -330,3 +351,34 @@ def input_arg(self):
         `Layer <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer>`__. Set to
         ``"inputs"``."""
         return self._input_arg
+
+    @staticmethod
+    def set_input_argument(input_name: Text = "inputs") -> None:
+        """
+        Set the name of the input argument.
+
+        Parameters
+        ----------
+        input_name (Text): Name of the input argument
```

Comment:
Since we use [Google-style](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html) docstrings here, this should be written as follows:
```suggestion
        Args:
            input_name (stre): Name of the input argument
```